db/migrate/20150312144232_migrate_websockets_setting.rb

```diff
@@ -0,0 +1,16 @@
+class MigrateWebsocketsSetting < ActiveRecord::Migration
+  def up
+    return unless old_value = Setting.find_by_name("websockets_encrypt")
+    new_value = (old_value == true || old_value == "true" || old_value == "auto") ? true : false
```

Comment:
`old_value` is a Setting isn't it, not the value of the setting?  I don't understand why you're comparing a Setting to true etc, because as far as I can see, it'll never be true.